docs/index.mdx

```diff
@@ -16,31 +16,173 @@ import {
   LangChainQuickStartCodeTabs,
   ConfigureEnvironmentCodeTabs,
   RunTreeQuickStartCodeTabs,
+  ConfigureSDKEnvironmentCodeTabs,
+  PythonSDKTracingCode,
+  TypeScriptSDKTracingCode,
 } from "@site/src/components/QuickStart";
 import { ClientInstallationCodeTabs } from "@site/src/components/ClientInstallation";
 import DocCardList from "@theme/DocCardList";
 
-# LangSmith
+# Getting started with LangSmith
 
 ## Introduction
 
-**LangSmith** is a platform for building production-grade LLM applications.
+**LangSmith** is a platform for building production-grade LLM applications. Monitor and evaluate your application in real-time, allowing you to ship quickly and with confidence. Use of LangChain is not necessary - LangSmith works on its own!
```

Comment:
Added explicit statement that LangSmith works without LangChain. However, you do need to import langchain to use runOnDataset in Typescript. Think it's still fine to call out here? I will add a linear ticket to remove any reference to LangChain from this page once we can evaluate in the typescript SDK directly. cc @hinthornw